website/src/app/dashboard/dashboard.component.html

```diff
@@ -29,7 +29,7 @@
 			<!-- Alert Count Column -->
 			<ng-container matColumnDef="alertCount">
 				<th mat-header-cell *matHeaderCellDef mat-sort-header>Active Alerts</th>
-				<td mat-cell *matCellDef="let row">{{row.alerts?.size}}</td>
+				<td mat-cell *matCellDef="let row">{{getActiveAlerts(row)}}</td> <!-- showing ths size of map-->
```

Comment:
Let's remove the comment here too
```suggestion
				<td mat-cell *matCellDef="let row">{{getActiveAlerts(row)}}</td>
```